### ISSUE TITLE
new opinionated project pnpm, dev won't start w/ typescript error

### ISSUE THREAD

**Issue by acroyear** [expressots/expressots#38]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current behavior

after 
```expressots new home-photos pnpm opinionated```
and
```pnpm run dev```
I get
```
[INFO] 15:09:23 ts-node-dev ver. 2.0.0 (using ts-node ver. 10.9.1, typescript ver. 4.9.5)
Compilation error in /Users/joe/Projects/home-photos/src/useCases/app/app.controller.ts
[ERROR] 15:09:24 ⨯ Unable to compile TypeScript:
src/useCases/app/app.controller.ts(14,9): error TS2322: Type 'Response<any, Record<string, any>>' is not assignable to type 'string'.
```

### Steps to reproduce

1. expressots new home-photos pnpm opinionated
2. pnpm run dev

### Expected behavior

something works without a typescript error

### Package version

1.5.0-dev

### Node.js version

20.4.0

### In which operating systems have you tested?

- [X] macOS
- [ ] Windows
- [ ] Linux

### Other

1.5.0-dev came up because OOB the cli seems to ask for '@latest'.

**Comment by acroyear** [comment]
after doing a any instead of string on that line, I get another:
```
[ERROR] 15:37:06 ⨯ Unable to compile TypeScript:
src/useCases/user/create/user-create.controller.ts(26,9): error TS2739: Type 'Response<any, Record<string, any>>' is missing the following properties from type 'ICreateUserResponseDTO': id, name, email, message
```

**Comment by rsaz** [comment]
Hi @acroyear 

The issue is mentioned here with a temporary solution: #36

**Comment by rsaz** [comment]
This issue is not happening on v1.4.2. Is only happening in the 1.5.0-dev which was the latest one. Please, update your package to the latest release which currently is 1.5.0. 

When you create new projects in the future it will get the latest and the latest is the current version v1.5.0 at the moment.

Let me know if it is all good.

**Comment by rsaz** [comment]
Problem resolved in v1.5.0
